services/ocr.service.ts

```diff
@@ -0,0 +1,27 @@
+import Axios from "axios";
+import { config } from "../config";
+
+export type DocumentType = "pdf" | "image" | "encoded";
+
+export class OcrService {
```

Comment:
Similar to `HtmlScraperService`... extracted logic for OCR stuff into this service... now it can be better unit tested and also reused for other parsers.

Reply:
similar suggestion :smile: